metaseq/hub_utils.py

```diff
@@ -700,23 +711,22 @@ def _filter_special(
         # scores is a 1D list of log-probability scores for those tokens (length seqlen)
         # distributions (optional) is a seqlen x vocab_size tensor corresponding to
         # the full distribution of predictions at each timestep
-
         output = []
         mask = []
-        for t, s in zip(tokens, scores):
+        for i, (t, s) in enumerate(zip(tokens, scores)):
             if t == pad_token:
                 # simply skip pads
                 mask.append(False)
                 continue
-            if t <= 3:
+            if t <= 3 and i > 0:
```

Comment:
This seems brittle, but probably making it less brittle requires too much glue.  We have asserts to make sure we're using the matching dictionary in streaming lm: https://github.com/facebookresearch/metaseq/blob/d2c75c2ca38c5a68e18c6c4aee2e06e78e2d7ab5/metaseq/tasks/streaming_language_modeling.py#L158-L166 - can we do something similar here?

Reply:
nvm i just did the right thing